apps/wing-console/console/server/src/router/app.ts

```diff
@@ -56,16 +52,16 @@ export const createAppRouter = () => {
     "app.wingfile": createProcedure.query(({ ctx }) => {
       return ctx.wingfile.split("/").pop();
     }),
+    /** @deprecated */
```

Comment:
We keep it for backwards compatibility, mostly because of possible version mismatches between Console frontend and Console backend. Not sure when or if it can happen, though.

Do you think we should just remove it?

Reply:
if for all current console usages there is no need for backward comp. - remove it.
Wing Cloud
Console (CLI)
Playground
VSCode

Reply:
Sure thing.